### PR TITLE
feat: add description property to type

### PR DESCRIPTION
**Pull request by kbaumzie**
Expose the description property for types

Contrib with @ankur-bambharoliya

### CODE REVIEW

File: test/converter.js [hunk 1]
```diff
@@ -112,13 +112,16 @@ async function testAttrbuteType (test, jsonType, graphQLType, options) {
     type: 'object',
     properties: {
       attribute: { type: jsonType }
-    }
+    },
+    description: 'Description'
   };
 
   const expectedType = `
+  "Description"
```
Comment (mdlavin) [inline test/converter.js]: Is this a valid syntax for GraphQL comments and documentation?

Comment (ankur-bambharoliya) [reply test/converter.js]: I think yes, refer this https://github.com/graphql/graphql-spec/issues/420

Comment (mdlavin) [reply test/converter.js]: Wow, I'm surprised I didn't know about that. The full spec is here: https://graphql.github.io/graphql-spec/draft/#sec-Descriptions

**Comment by mdlavin** [comment]
Thanks for your contribution! I released @lifeomic/json-schema-to-graphql-types@1.4.0 with your fix

**Pull request merged**